app/models/migration/ecf/participant_profile.rb

```diff
@@ -0,0 +1,12 @@
+module Migration::Ecf
+  class ParticipantProfile < BaseRecord
+    self.inheritance_column = nil
+
+    belongs_to :teacher_profile
+    belongs_to :schedule, class_name: "Finance::Schedule"
+    has_one :user, through: :teacher_profile
+    has_one :npq_application, foreign_key: :id
+
+    default_scope { where("participant_profiles.type = ?", "ParticipantProfile::NPQ") }
```

Comment:
Out of interest, how come SQL over just using AR?

```suggestion
    default_scope { where(type: "ParticipantProfile::NPQ") }
```